chsdi/views/layers.py

```diff
@@ -160,6 +160,56 @@ def insertValueAt(field, attrName, value):
     return {'id': layerId, 'name': params.translate(layerId), 'fields': fields}
 
 
+@view_config(route_name='faqlist', renderer='jsonp')
+def faqlist(request):
+    params = LayersParams(request)
+    params.geodataStaging = 'prod'
+    translations = {}
+
+    # That there is a geometry behind
+    queryableLayers = []
+    # That there is a tooltip
+    tooltipLayers = []
+    # That you can search in layer search
+    searchableLayers = []
+    # That you need to pay for
+    chargeableLayers = []
+    # Free layers
+    notChargeableLayers = []
+
+    query = params.request.db.query(LayersConfig)
+    for layer in get_layers_config_for_params(params, query, LayersConfig):
+        k = layer.keys().pop()
+        l = layer.values().pop()
+        if 'parentLayerId' not in l and not k.endswith('_3d'):
+            if k not in translations:
+                translations[k] = request.translate(k)
+            if 'queryable' in l and l['queryable']:
+                queryableLayers.append(l)
+            if 'tooltip' in l and l['tooltip']:
+                tooltipLayers.append(l)
+            if 'searchable' in l and l['searchable']:
+                searchableLayers.append(l)
+            if 'chargeable' in l and l['chargeable']:
+                chargeableLayers.append(l)
+            if 'chargeable' in l and not l['chargeable']:
+                notChargeableLayers.append(l)
+
+    queryableLayers.sort()
+    tooltipLayers.sort()
+    searchableLayers.sort()
+    chargeableLayers.sort()
+    notChargeableLayers.sort()
+    return {
+        'translations': translations,
+        'queryableLayers': queryableLayers,
```

Comment:
Like this it was returning null, probably because of the comma exactly after `.sort()`